Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:2.1.2
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
```

Comment:
Why do we need nodejs?

Reply:
Rails requires a javascript runtime for things like transpiling coffeescript and minifying, and the Docker docs recommended nodejs so that's what I used